### PR TITLE
Personal Portfolio Desktop

### PR DESCRIPTION
**Pull request by dialite**
This branch contains a responsive web page content aimed at designing a desktop view from the mobile first page. It was written in html and css.

### CODE REVIEW

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @dialite 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-My Personal Portfolio(Mobile)-Microverse
+My Personal Portfolio(Desktop)-Microverse
 
 > My Portfolio
-This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage.
+This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage. It is the desktop version.
 
 ## Built With
 - HTML
```
Comment (Kajal19-del) [inline README.md:9]: Nice work! You did it 💯 🥂

File: README.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ This is a my first phase project on my personal portfolio development with micro
...
  To Access my project locally please follow the following instructions;
 1. Create a folder on your computer where you will clone the project
-2. Enter this url: https://github.com/dialite/Portfolio-Mobile/tree/updates in your web browser.
+2. Enter this url: https://github.com/dialite/Portfolio-Mobile/tree/portfolio-desktop in your web browser.
 3. Once opened, move your cursor to the top left of the project page to select 'code' a green code button will be visible on the righthand corner showing download    options.
 5. Select download Zip option from drop down menu to your desired folder on your computer.
 6. Once the download is complete you will be able to access my project locally.
 7. Also, you can navigate to your folder using a console.
-8. Then run git clone https://github.com/dialite/Portfolio-Mobile/tree/updates
+8. Then run git clone https://github.com/dialite/Portfolio-Mobile/tree/portfolio-desktop
```
Comment (Kajal19-del) [inline README.md:26]: Thanks for putting instruction there, you did very well 💯 🥇

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-My Personal Portfolio(Mobile)-Microverse
+My Personal Portfolio(Desktop)-Microverse
 
 > My Portfolio
-This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage.
+This is a my first phase project on my personal portfolio development with microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage. It is the desktop version.
```
Comment (ivanmvh) [inline README.md:7]: You can put a link to github pages then you can show your work inmediately to the word.

File: README.md [hunk 4]
```diff
@@ -16,15 +16,15 @@ This is a my first phase project on my personal portfolio development with micro
...
  To Access my project locally please follow the following instructions;
 1. Create a folder on your computer where you will clone the project
-2. Enter this url: https://github.com/dialite/Portfolio-Mobile/tree/updates in your web browser.
+2. Enter this url: https://github.com/dialite/Portfolio-Mobile/tree/portfolio-desktop in your web browser.
 3. Once opened, move your cursor to the top left of the project page to select 'code' a green code button will be visible on the righthand corner showing download    options.
 5. Select download Zip option from drop down menu to your desired folder on your computer.
 6. Once the download is complete you will be able to access my project locally.
 7. Also, you can navigate to your folder using a console.
-8. Then run git clone https://github.com/dialite/Portfolio-Mobile/tree/updates
+8. Then run git clone https://github.com/dialite/Portfolio-Mobile/tree/portfolio-desktop
```
Comment (Kajal19-del) [inline README.md:24]: You did it. Thank You. 💯 🥇

**Review by Kajal19-del — COMMENTED** [review]
# You completed all the given instructions. Cheers 🥂 🥳

File: README.md (hunk 4 above)
Comment (Zia123456) [inline README.md:24]: # Good readme file
your readme file looks professional.

# optional
If you add the license that you use for this project, your readme file will be perfect.
if your readme file has a demo link of your project that will be very nice.

**Review by Zia123456 — COMMENTED** [review]
Nice work.

**Pull request merged**